core/src/components/datetime-button/test/multiple/datetime-button.e2e.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from '@playwright/test';
```

Comment:
Could we add an `index.html` file too, to make it easier to play with the behavior in real time?

Reply:
Added in https://github.com/ionic-team/ionic-framework/pull/25971/commits/0780fc461597a1097912cb3b96d421710d446d87